diff '--color=auto' -ur a/eglew.h b/eglew.h
--- a/eglew.h	2026-04-01 23:08:35.280971262 +0200
+++ b/eglew.h	2026-04-01 23:07:54.807219349 +0200
@@ -107,7 +107,7 @@
 #include <EGL/eglplatform.h>
 
 #ifndef GLEW_INCLUDE
-#  include <GL/glew.h>
+#  include "external/glew/glew.h"
 #else
 #  include GLEW_INCLUDE
 #endif
diff '--color=auto' -ur a/glew.c b/glew.c
--- a/glew.c	2026-04-01 23:08:36.529963607 +0200
+++ b/glew.c	2026-04-01 23:07:57.764201222 +0200
@@ -31,7 +31,7 @@
 */
 
 #ifndef GLEW_INCLUDE
-#  include <GL/glew.h>
+#  include "external/glew/glew.h"
 #else
 #  include GLEW_INCLUDE
 #endif
@@ -48,7 +48,7 @@
 #    undef GLEW_APIENTRY_DEFINED
 #  endif
 #elif defined(GLEW_EGL)
-#  include <GL/eglew.h>
+#  include "external/glew/eglew.h"
 #elif defined(_WIN32)
 /*
  * If NOGDI is defined, wingdi.h won't be included by windows.h, and thus
@@ -58,9 +58,9 @@
 #  if defined(NOGDI)
 #    undef NOGDI
 #  endif
-#  include <GL/wglew.h>
+#  include "external/glew/wglew.h"
 #elif !defined(__ANDROID__) && !defined(__native_client__) && !defined(__HAIKU__) && (!defined(__APPLE__) || defined(GLEW_APPLE_GLX))
-#  include <GL/glxew.h>
+#  include "external/glew/glxew.h"
 #endif
 
 #include <stddef.h>  /* For size_t */
diff '--color=auto' -ur a/glxew.h b/glxew.h
--- a/glxew.h	2026-04-01 23:08:38.874949236 +0200
+++ b/glxew.h	2026-04-01 23:08:00.392185113 +0200
@@ -100,7 +100,7 @@
 #include <X11/Xmd.h>
 
 #ifndef GLEW_INCLUDE
-#  include <GL/glew.h>
+#  include "external/glew/glew.h"
 #else
 #  include GLEW_INCLUDE
 #endif
