diff -ur a/eglew.h b/eglew.h
--- a/eglew.h	2017-08-20 12:01:35.410583684 +0200
+++ b/eglew.h	2017-08-20 12:02:48.134928758 +0200
@@ -106,7 +106,7 @@
 #include <KHR/khrplatform.h>
 #include <EGL/eglplatform.h>
 
-#include <GL/glew.h>
+#include "glew/glew.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -ur a/glew.c b/glew.c
--- a/glew.c	2017-08-20 12:01:29.324638491 +0200
+++ b/glew.c	2017-08-20 12:04:10.590186200 +0200
@@ -31,7 +31,7 @@
 */
 
 #ifndef GLEW_INCLUDE
-#include <GL/glew.h>
+#include "glew/glew.h"
 #else
 #include GLEW_INCLUDE
 #endif
@@ -40,7 +40,7 @@
 #  define GLAPI extern
 #  include <GL/osmesa.h>
 #elif defined(GLEW_EGL)
-#  include <GL/eglew.h>
+#  include "glew/eglew.h"
 #elif defined(_WIN32)
 /*
  * If NOGDI is defined, wingdi.h won't be included by windows.h, and thus
@@ -50,9 +50,9 @@
 #  if defined(NOGDI)
 #    undef NOGDI
 #  endif
-#  include <GL/wglew.h>
+#  include "glew/wglew.h"
 #elif !defined(__ANDROID__) && !defined(__native_client__) && !defined(__HAIKU__) && (!defined(__APPLE__) || defined(GLEW_APPLE_GLX))
-#  include <GL/glxew.h>
+#  include "glew/glxew.h"
 #endif
 
 #include <stddef.h>  /* For size_t */
@@ -16218,7 +16218,7 @@
 
   if (GLEW_VERSION_3_0)
   {
-    GLint n = 0;
+    GLint nExt = 0;
     GLint i;
     PFNGLGETINTEGERVPROC getIntegerv;
     PFNGLGETSTRINGIPROC getStringi;
@@ -16232,12 +16232,12 @@
     #endif
 
     if (getIntegerv)
-      getIntegerv(GL_NUM_EXTENSIONS, &n);
+      getIntegerv(GL_NUM_EXTENSIONS, &nExt);
 
     /* glGetStringi is OpenGL 3.0 */
     getStringi = (PFNGLGETSTRINGIPROC) glewGetProcAddress((const GLubyte*)"glGetStringi");
     if (getStringi)
-      for (i = 0; i<n; ++i)
+      for (i = 0; i<nExt; ++i)
       {
         ext = (const char *) getStringi(GL_EXTENSIONS, i);
 
diff -ur a/glxew.h b/glxew.h
--- a/glxew.h	2017-08-20 12:01:35.410583684 +0200
+++ b/glxew.h	2017-08-20 12:02:48.137928731 +0200
@@ -98,7 +98,7 @@
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
 #include <X11/Xmd.h>
-#include <GL/glew.h>
+#include "glew/glew.h"
 
 #ifdef __cplusplus
 extern "C" {
